.github/pull_request_template.md

```diff
@@ -5,4 +5,4 @@
 - [ ] Documentation added for any new functionality
 - [ ] Entries added to the respective `CHANGELOG.md` for any new functionality
 - [ ] Feature table on [`README.md`](../README.md#roadmap) updated for any listed functionality
-- [ ] Regenerated documentation with `quint docs` if `builtin.qnt` was changed
```

Comment:
Do we actually need a checklist item for this if it is already being checked by CI?

Reply:
Makes sense.